### PR TITLE
CORPIT-1710 Add a script to reset a yubikey

### PR DESCRIPTION
**Pull request by daisukixci**
Sometimes Corp IT get yubikeys back, then need to reset them, this script is here to facilitate that

### CODE REVIEW

File: reset.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            echo "Cancelling"
+            return 1
+    esac
+}
+
+reset_device() {
+    local serial
+    serial="$1"
+
+    /usr/local/bin/ykman --device "$(serial)" otp delete 1 -f
```
Comment (trishankatdatadog) [inline reset.sh:25]: I believe we can refer to the `YKMAN` env var in `env.sh` throughout, instead of calling directly?

File: reset.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    /usr/local/bin/ykman --device "$(serial)" otp delete 2 -f
+    /usr/local/bin/ykman --device "$(serial)" oath reset -f
+    /usr/local/bin/ykman --device "$(serial)" openpgp reset -f
+    /usr/local/bin/ykman --device "$(serial)" piv reset -f
+    /usr/local/bin/ykman --device "$(serial)" fido reset -f
+}
+
+# Don't have you personnal yubikey plugged in
```
Comment (trishankatdatadog) [inline reset.sh:33]:
```suggestion
# Don't have you personal yubikey plugged in
```

File: reset.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+# to run me `bash yubispam.sh`
```
Comment (trishankatdatadog) [inline reset.sh:34]: Sorry, what does this mean? Where is this script?

File: reset.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+yubikeys=$(/usr/local/bin/ykman list --serials)
+select serial in all $yubikeys cancel; do
+    echo "You choose $serial"
```
Comment (trishankatdatadog) [inline reset.sh:37]:
```suggestion
    echo "You chose $serial"
```

File: reset.sh (hunk 3 above)
Comment (daisukixci) [reply reset.sh:34]: This script was first a script to remove the yubispam issue within corpit. I removed the comment as it's not true anymore

File: reset.sh (hunk 1 above)
Comment (daisukixci) [reply reset.sh:25]: Done

File: reset.sh (hunk 4 above)
Comment (daisukixci) [reply reset.sh:37]: done

**Comment by trishankatdatadog** [comment]
Great, thanks @daisukixci! Could we please just add this to a subsection in the README? 🙂

File: README.md [hunk 5]
```diff
@@ -237,6 +237,17 @@ and export it to both YubiKey, and keep a copy on disk:
 
 **Table 6**: Add these lines to `~/.bashrc`.
 
+## Reset your yubikey
+
+If you need/want to reset your/serveral yubikey/s, you can use the reset.sh. The script look for every plugged yubikey
```
Comment (trishankatdatadog) [inline README.md:242]:
```suggestion
If you need to reset YubiKeys, you may use the following script. The script looks for every plugged YubiKey,
```

File: README.md [hunk 6]
```diff
@@ -237,6 +237,17 @@ and export it to both YubiKey, and keep a copy on disk:
...
+then shows a menu to reset one specific key or all of them.
```
Comment (trishankatdatadog) [inline README.md:243]:
```suggestion
and shows a menu to reset one specific key, or all of them.
```

File: README.md [hunk 7]
```diff
@@ -237,6 +237,17 @@ and export it to both YubiKey, and keep a copy on disk:
...
+**Please read and follow all of the instructions carefully. YOU WILL NOT BE ABLE TO RETRIEVE KEYS/DATA FROM THE YUBIKEY**
```
Comment (trishankatdatadog) [inline README.md:244]:
```suggestion
**Please read and follow all of the instructions carefully. YOU WILL NOT BE ABLE TO RETRIEVE KEYS/DATA FROM THE YUBIKEY AFTER COMPLETION**
```

**Pull request merged**